packages/test/jest.test.js

```diff
@@ -177,3 +169,15 @@ test("Error: method not found", async () => {
     )
   ).toMatchSnapshot();
 });
+
+test("delete entry", async () => {
+  await call("deleteEntry", { id: entry1.id });
+  await call("deleteEntry", { id: entry2.id });
+  await call("deleteEntry", { id: entry3.id });
+
+  const response = await call("listEntries");
+
+  expect(response.includes(entry => entry.id === entry1.id)).toBe(false);
+  expect(response.includes(entry => entry.id === entry2.id)).toBe(false);
+  expect(response.includes(entry => entry.id === entry3.id)).toBe(false);
+});
```

Comment:
I do not know if it's useful but you told me yesterday that we have to run the tests without relaunching the server (with jest --watch). I don't know if that is what you want me to do.